src/main/groovy/wooga/gradle/build/unity/UnityBuildPlugin.groovy

```diff
@@ -42,6 +44,8 @@ import wooga.gradle.unity.utils.GenericUnityAssetFile
 class UnityBuildPlugin implements Plugin<Project> {
 
     static final String EXTENSION_NAME = "unityBuild"
+    static final String DEFAULT_EXPORT_METHOD = "Wooga.UnifiedBuildSystem.Editor.BuildEngine.BuildFromEnvironment"
```

Comment:
@Azurelol 
I said it before. I would like to reduce the amount of times we use the term `build`. The context for this feature has slightly moved to be more generic execution. I would suggest something down the line:
`Wooga.UnifiedBuildSystem.Editor.Engine.ExecuteWithEnvironment` or similar. Also the naming of `BuildEngine` would suggest that we have multiple engines in UBS ;)

Reply:
Also the name of the variable becomes slightly confusing. In the beginning we refered to the tasks that triggered the export of the external projects as `export` tasks. That is still true for the build player pipelines but no longer for all other tasks. Because they don't have or maybe won't ever export anything.

Reply:
Renaming the build engine to engine is something I could get behind, as it as you say: it's a task execution system.

I agree with avoiding the use of export in the nomenclature as it sets an expectation. What about DEFAULT_METHOD_NAME?

Reply:
I think that `DEFAULT_METHOD_ NAME` is too generic and it doesn't say anything about the method. As this string is only used once, I'll just remove the variable and put the string directly into place. I think it makes more clear its purpose when it is in its own context.